pkgs/by-name/lw/lwk/package.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, fetchFromGitHub, rustPlatform, pkg-config, udev }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "lwk";
+  version = "0.2.0";
+  cargoBuildFlags = "--bin lwk_cli";
```

Comment:
nit: I would move this down to line 16 or so, simply because it's a bit obscured up here